userguide/image_classifier/README.md

```diff
@@ -30,6 +30,43 @@ data.save('cats-dogs.sframe')
 data.explore()
 ```
 
+#### Annotating Data
+
+If you only have images without corresponding annotations, you can 
+use the annoation utility built into the image_classifier. An example
+of its usage is shown below with the **PetImages** subfolder above.
```

Comment:
Shown below with the subfolder above? Then the code starts with the code above, again? That's a bit confusing. Let's just say "shown below", then skip to the relevant parts of the code and assume the user has already typed in the above code.